hal/api/AnalogOut.h

```diff
@@ -122,6 +122,10 @@ class AnalogOut {
     }
 #endif
 
+    virtual ~AnalogOut() {
+        // Do nothing
```

Comment:
Just wanted to indicate that the code was not forgotten accidentally.  Want me to remove it?